ui/admin/tests/acceptance/storage-buckets/selectors.js

```diff
@@ -0,0 +1,30 @@
+export const SAVE_BTN = '[type=submit]';
+export const CANCEL_BTN = '.rose-form-actions [type=button]';
+export const EDIT_BTN = '.rose-form-actions [type=button]';
+export const FIELD_NAME = '[name=name]';
+export const FIELD_NAME_VALUE = 'Random name';
+export const FIELD_SCOPE = '[name=scope]';
+export const FIELD_PLUGIN_TYPE = '[name=plugin_type]';
+export const FIELD_PLUGIN_TYPE_MINIO = '[value=minio]'; // Provider
+export const FIELD_ENDPOINT_URL = '[name=endpoint_url]';
+export const FIELD_ENDPOINT_URL_VALUE = 'http://www.hashicorp.com';
+export const FIELD_BUCKET_NAME = '[name=bucket_name]';
+export const FIELD_BUCKET_NAME_VALUE = 'Test Bucket name';
+export const FIELD_BUCKET_NAME_ERROR = '#error-bucket_name';
+export const FIELD_BUCKET_PREFIX = '[name=bucket_prefix]';
+export const FIELD_REGION = '[name=region]';
+export const GROUP_CREDENTIAL_TYPE = '[name=credential_type]';
+export const FIELD_STATIC_CREDENTIAL = '[value=static]';
+export const FIELD_DYNAMIC_CREDENTIAL = '[value=dynamic]';
+export const FIELD_ROLE_ARN = '[name=role_arn]';
+export const FIELD_ROLE_ARN_VALUE = 'test-arn-id';
+export const FIELD_ACCESS_KEY = '[name=access_key_id]';
+export const FIELD_ACCESS_KEY_EDIT_BTN = '#access_key_id [type=button]';
+export const FIELD_ACCESS_KEY_VALUE = '45$#@%!783528';
+export const FIELD_SECRET_KEY = '[name=secret_access_key] ';
+export const FIELD_SECRET_KEY_VALUE = '87^%$#@^86382';
+export const FIELD_SECRET_KEY_EDIT_BTN = '#secret_access_key [type=button]';
+export const FIELD_WORKER_FILTER = '[name=worker_filter]';
```

Comment:
Curious what your thoughts are on potentially making these more generic with an approach like I have below? I feel like I have already talked myself out of this idea but I wanted to run it by you to see what you thought.
```
export const FIELD_NAME = (name) => `[name=${name}]`;

assert.dom(FIELD_NAME('worker_filter')).exist();
```

Reply:
@cameronperera 
I do think it's a very nice approach, I am not opposed at all to a more dynamic approach as you suggest. Maybe we can validate the idea at a tech time with others opinion?

Personally, I will bet to start with a static approach, a constant being defined, rather than being a function to be called. So we can simplify it to focus on set first naming conventions and selecting patterns. 
Later on add more generic  and dynamic improvements as you are suggesting?

Just to be clear, not opposing at all, I do think it's a nice one.